examples/new_project_templates/single_cpu_template.py

```diff
@@ -9,12 +9,12 @@
 from pytorch_lightning.models.trainer import Trainer
 from pytorch_lightning.callbacks import EarlyStopping, ModelCheckpoint
 
+from .lightning_module_template import LightningTemplateModel
```

Comment:
will this relative import also break here?

Reply:
it is now updated and use full import path